proxy/proxy/client.go

```diff
@@ -225,6 +228,15 @@ func (c *Client) Dial(instance string) (net.Conn, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// Refresh cert 5 minutes before it expires.
+	timer := time.NewTimer(cfg.Certificates[0].Leaf.NotAfter.Sub(time.Now()) - refreshCertBuffer)
```

Comment:
Since you're not reusing the time.Timer, it's simpler to use `time.After` instead.

Reply:
Done.